config.py.example

```diff
@@ -8,3 +8,4 @@ store_path = 'profile/'
 manager_accounts = ('') # name here accounts responsible for device keys verifiacation
 proxy = 'http://your.proxy:8080'
 user_agent = 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) QtWebEngine/5.14.1 Chrome/77.0.3865.129 Safari/537.36'
+feeder_period = 300 # interval between updates
```

Comment:
I mean I know it's just an example, but I think more meaningful default could be ~30 minutes, not 5 mins.

Reply:
It's more of a template rather than an example actually, so we'd like to have sane value here, yep. So what is specifically wrong with getting RSS updates every 5 minutes?

Reply:
yes, 5 minutes is waay too frequent. 30 minutes or 1 hour would be more appropriate.